resources/build.gradle.kts

```diff
@@ -7,7 +7,7 @@ import software.aws.toolkits.gradle.resources.ValidateMessages
 plugins {
     id("toolkit-kotlin-conventions")
     id("toolkit-testing")
-    id("de.undercouch.download")
+    id("de.undercouch.download") version "5.2.1"
```

Comment:
We dont usually specify plugin version numbers, would this break current users?

Reply:
no this is a build dep